src/core/js/app.js

```diff
@@ -121,14 +132,11 @@ require([
     // This function is called when the config model triggers 'configModel:loadCourseData'
     // Once the config model is loaded get the course files
     // This enables plugins to tap in before the course files are loaded & also to change the default language
-    Adapt.loadCourseData = function(isLanguageChange) {
+    Adapt.loadCourseData = function(language) {
         Adapt.on('adaptCollection:dataLoaded courseModel:dataLoaded', function() {
-            Adapt.checkDataIsLoaded(isLanguageChange);
+            Adapt.checkDataIsLoaded(language);
         });
 
-        // All code that needs to run before adapt starts should go here
-        var language = Adapt.config.get('_activeLanguage');
```

Comment:
As a consequence of removing this declaration, `language` will be undefined on first load.

Would you mind terribly restoring this and then renaming our passed-through `language` variable as `newLanguage` so it's more obvious what it represents?